Curve-finance-suite/New-Fee/README.md

```diff
@@ -0,0 +1,26 @@
+# New Fee Agent
+
+## Description
+
+This agent detects a change in fees to a Curve BasePool or MetaPool by the Curve DAO
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+Describe each of the type of alerts fired by this agent
+
+- CURVE-2
+  - Fired when `set_new_fee` function is called in the Curve DAO contract
+  - Severity is always set to "info"
+  - Type is always set to "info"
+  - Metadata contains:
+    - `affected_pool`: The address of the pool that emitted the event
```

Comment:
I think this is correct, but it would be more exact to say something like `_pool argument used in the call`, because you are not checking any event, you are looking for method calls.

Reply:
Addressed in latest commit